src/LanguageServer.Engine/Handlers/ConfigurationHandler.cs

```diff
@@ -139,11 +114,11 @@ object IRegistration<object>.GetRegistrationOptions()
         /// <returns>
         ///     An <see cref="IDisposable"/> representing the log-context scope.
         /// </returns>
-        IDisposable BeginOperation(string operationName)
+        private static IDisposable BeginOperation(string operationName)
         {
-            if (String.IsNullOrWhiteSpace(operationName))
+            if (string.IsNullOrWhiteSpace(operationName))
```

Comment:
This also a conscious choice, BTW - `string` for data-type, `String` for static type-member access.

Reply:
I noticed that choice of `string` vs `String` seems to have some pattern, but I wasn't sure what exactly. I relied on the fact that by default .NET code style recommendations are to use keywords rather than type names and there was no specific code style rule e.g. in `.editorconfig` to prevent it. I am not sure though if there is a rule for this specific case, I am only aware that you can either choose to preffer keywords or type names in all places. When you take closer look at code you can put together a list of changes that you don't like in the PR and I will revert them

Reply:
No problem - thanks being open to discussing this stuff, I appreciate it. And you're right I should probably have fleshed out the editorconfig a while ago (as you say without adequate documentation it's hard for someone contributing to know why a given thing is the way it is and they just have to guess) but other stuff just got in the way. I'll see if I can copy over the editorconfig from another of my projects (they're mostly pretty consistent) as a starting point and maybe we can look at it together and see what makes sense and what doesn't?

I'll try to get that done first thing tomorrow morning 🙂